app/dao/notifications_dao.py

```diff
@@ -374,19 +397,19 @@ def get_notifications(filter_dict=None):
 
 @statsd(namespace="dao")
 def get_notifications_for_service(
-        service_id,
-        filter_dict=None,
-        page=1,
-        page_size=None,
-        count_pages=True,
-        limit_days=None,
-        key_type=None,
-        personalisation=False,
-        include_jobs=False,
-        include_from_test_key=False,
-        older_than=None,
-        client_reference=None,
-        include_one_off=True
+    service_id,
+    filter_dict=None,
+    page=1,
+    page_size=None,
+    count_pages=True,
+    limit_days=None,
+    key_type=None,
+    personalisation=False,
+    include_jobs=False,
+    include_from_test_key=False,
+    older_than=None,
+    client_reference=None,
+    include_one_off=True
```

Comment:
For the last line of a function signature?

Reply:
Yeah, [PEP8](https://peps.python.org/pep-0008/#when-to-use-trailing-commas). Otherwise if you add a new parameter it changes two lines rather than one.

> The pattern is to put each value (etc.) on a line by itself, always adding a trailing comma, and add the close parenthesis/bracket/brace on the next line. However it does not make sense to have a trailing comma on the same line as the closing delimiter (except in the above case of singleton tuples)